cognite/neat/_session/_prepare.py

```diff
@@ -133,6 +132,32 @@ def relationships_as_edges(self, min_relationship_types: int = 1, limit_per_type
         transformer = RelationshipAsEdgeTransformer(min_relationship_types, limit_per_type)
         self._state.instances.store.transform(transformer)
 
+    def convert_data_type(self, source: tuple[str, str], *, convert: Callable[[Any], Any]) -> None:
```

Comment:
Note that the callable here is a deviation from requiring only primitively types as input in session method.

Reply:
see my comment on defaulting to conversion method